coralproject/talk
# [CORL-1317] Warning Messaging Improvements

## What does this PR do?

When a user is warned after they've loaded a page, future actions will now show a message with information to redirect the user to the top to acknowledge the warning. For reactions, it will scroll the user to the message instead of showing a warning currently.

This also corrects some GraphQL `@auth` directives to narrow the scope and fix some issues with permitted statuses.



## What changes to the GraphQL/Database Schema does this PR introduce?

Many auth directives were removed/added to conform to correct usages within the application. Review the files changed on this PR for all changes introduced.



## How do I test this PR?



1. Load the stream as a un-warned user.
2. In another tab, warn the user.
3. Try to react, reply, and flag a comment as the warned user and see correct behaviour.
